.travis.yml

```diff
@@ -38,10 +36,11 @@ install:
   && printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh && cd $TRAVIS_BUILD_DIR;
   fi
 - printf '\ny\n' | $HOME/gcloud/google-cloud-sdk/bin/gcloud components install app-engine-python
+- alias wct="$TRAVIS_BUILD_DIR/client/node_modules/web-component-tester/bin/wct"
 script:
 - grunt lint
 - python tests.py $HOME/gcloud/google-cloud-sdk
-- xvfb-run wct --skip-plugin sauce
+- xvfb-run $TRAVIS_BUILD_DIR/client/node_modules/web-component-tester/bin/wct --skip-plugin sauce
 - "(cd analysis; npm test)"
```

Comment:
I see. I think people are familiar with the existing one, so I'll leave it for now.

`yarn` build is significantly faster and is deterministic, so its worth the switch here.